mark-griffith/short-response.md

```diff
@@ -0,0 +1,85 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+      Nothing because its missing a `
+
+      But seriously this code first initializes and declares a variable of singer with the string
+      Omar Apollo, the it logs the phrase my favorite singer is with the value of singer using a template literal.
+      "My favorite singer is Omar Apollo"
+      
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+      At first this runs the same way as tomorrow, 
```

Comment:
Not quite. This is an important one because we will use `const` a LOT. Run this in the console and see what you get. Report back with what and why.